packages/algob/src/lib/account.ts

```diff
@@ -71,3 +71,50 @@ export function mkAccountIndex (accountList: Account[]): Accounts {
   }
   return out;
 }
+
+function validateAlgobAccounts (algobAccounts: AlgobAccount[]): void {
+  for (const account of algobAccounts) {
+    if (account.name === undefined) {
+      throw new BuilderError(ERRORS.ACCOUNT.NAME_MISSING,
+        { errors: 'account name is missing in ' + JSON.stringify(account) });
+    }
+    if (account.mnemonic === undefined) {
+      throw new BuilderError(ERRORS.ACCOUNT.MNEMONIC_MISSING,
+        { errors: 'mnemonic string is missing in ' + JSON.stringify(account) });
+    }
+    if (account.name === "") {
+      throw new BuilderError(ERRORS.ACCOUNT.NAME_EMPTY,
+        { errors: 'account name is empty in ' + JSON.stringify(account) });
+    }
+    if (account.mnemonic === "") {
+      throw new BuilderError(ERRORS.ACCOUNT.MNEMONIC_EMPTY,
+        { errors: 'mnemonic string is empty in ' + JSON.stringify(account) });
+    }
+  }
+}
+
+export function loadFromEnv (): Account[] {
+  var algobAccountsString = process.env.ALGOB_ACCOUNTS;
+  if (algobAccountsString) {
+    var algob: AlgobAccount[] = [];
+    try {
+      algob = JSON.parse(algobAccountsString);
+    } catch (error) {
+      console.log(error);
```

Comment:
1. use `console.error` (below as well).
2. You should wrap here with `MALFORMED` error.